makefile.shade

```diff
@@ -86,14 +86,35 @@ var FULL_VERSION = '${PRODUCT_VERSION + "-" + E("DOTNET_BUILD_VERSION")}'
         sourceFiles.Add(Path.Combine(libuvRoot, "src/unix/linux-inotify.c"));
         sourceFiles.Add(Path.Combine(libuvRoot, "src/unix/linux-syscalls.c"));
 
+        var sources = string.Join(" ", sourceFiles);
+
         var outputDir = Path.Combine(ROOT, "src/libuv/bin/linux");
         var outputPath = Path.Combine(outputDir, "libuv.so");
         Directory.CreateDirectory(outputDir);
 
-        var sources = string.Join(" ", sourceFiles);
+        var armOutputDir = Path.Combine(ROOT, "src/libuv/bin/linux-arm");
+        var armOutputPath = Path.Combine(armOutputDir, "libuv.so");
+        Directory.CreateDirectory(armOutputDir);
+
+        var arm64OutputDir = Path.Combine(ROOT, "src/libuv/bin/linux-arm64");
+        var arm64OutputPath = Path.Combine(arm64OutputDir, "libuv.so");
+        Directory.CreateDirectory(arm64OutputDir);
+
+        var armelOutputDir = Path.Combine(ROOT, "src/libuv/bin/linux-armel");
+        var armelOutputPath = Path.Combine(armelOutputDir, "libuv.so");
+        Directory.CreateDirectory(armelOutputDir);
 
         Exec(CLANG,
             string.Format("{0} -lm -pthread -ldl -lrt -fPIC -shared -o {1}  -I{2}/include -I{2}/src -Wall -Wextra -Wno-unused-parameter -Wstrict-aliasing -g --std=gnu89 -pedantic -D_GNU_SOURCE -D_BUILDING_UV_SHARED=1", sources, outputPath, libuvRoot));
+
+        Exec(CLANG,
+            string.Format("{0} -target armv7a-linux-gnueabihf -B/usr/arm-linux-gnueabihf/bin/ -B/usr/arm-linux-gnueabihf/lib/ -isystem /usr/arm-linux-gnueabihf/include -lm -pthread -ldl -lrt -fPIC -shared -o {1}  -I{2}/include -I{2}/src -Wall -Wextra -Wno-unused-parameter -Wstrict-aliasing -g --std=gnu89 -pedantic -D_GNU_SOURCE -D_BUILDING_UV_SHARED=1", sources, armOutputPath, libuvRoot));
```

Comment:
Yes, I think so. The CoreFX and CoreCLR are linked to `armv7-linux-gnueabihf` and `armv7-linux-gnueabi` as target for building arm and armel. It will also work in `armv7` using `armv5`, but it is probably better to build it in the same way as CoreCLR and CoreFX. When I modified the target parameter, it builds well and it works well too.